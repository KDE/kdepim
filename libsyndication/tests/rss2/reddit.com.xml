<rss version="2.0" xmlns:dc="http://purl.org/dc/elements/1.1/"><channel>
<!--
    source: reddit.com
    Properties:
     - dc:date timestamp with millis
     - title: plain text
     - description: escaped HTML
    -->
        <title>reddit: what&#39;s new online</title>
        <link>http://reddit.com/</link>
        <description>The latest stories, voted on by users like you.</description>
          <item>
            <title> South Park Throws Down Gauntlet to Scientology, Declare themselves &quot;servants of the dark lord Xenu.&quot;</title>
            <link>http://reddit.com/goto?rss=true&amp;id=38p1</link>
            <dc:date>2006-03-17T20:38:18.009970-05:00</dc:date>
              <description>&lt;a href="http://www.variety.com/article/VR1117939918?categoryId=14&amp;cs=1"&gt;[link]&lt;/a&gt;&lt;a href="http://reddit.com/info?id=38p1"&gt;[more]&lt;/a&gt;</description>
          </item>
</channel></rss>
