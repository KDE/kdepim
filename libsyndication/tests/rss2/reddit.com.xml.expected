# Feed begin ######################
title: #reddit: what's new online#
link: #http://reddit.com/#
description: #The latest stories, voted on by users like you.#
# Item begin ######################
id: #hash:90e5a3bfeb84b99d0f121d4575ea991f#
title: #South Park Throws Down Gauntlet to Scientology, Declare themselves &quot;servants of the dark lord Xenu.&quot;#
link: #http://reddit.com/goto?rss=true&id=38p1#
description: #<a href="http://www.variety.com/article/VR1117939918?categoryId=14&cs=1">[link]</a><a href="http://reddit.com/info?id=38p1">[more]</a>#
datePublished: #Sat Mar 18 02:38:18 2006#
dateUpdated: #Sat Mar 18 02:38:18 2006#
# Item end ########################
# Feed end ########################

